spec/lib/auth0/api/v2/tickets_spec.rb

```diff
@@ -7,7 +7,8 @@
   context '.post_email_verification' do
     it { expect(@instance).to respond_to(:post_email_verification) }
     it 'expect client to send post to /api/v2/tickets/email-verification with body' do
-      expect(@instance).to receive(:post).with('/api/v2/tickets/email-verification', user_id: 'user_id', result_url: nil)
+      expect(@instance).to receive(:post).with('/api/v2/tickets/email-verification', user_id: 'user_id',
```

Comment:
Probably want one more for a non-nil `ttl_sec` ... and another to confirm what happens if a non-Integer is used for `ttl_sec`